README.md

```diff
@@ -111,37 +118,36 @@ Commands:
 
 Options:
 
-  -v, --version     显示版本信息
```

Comment:
removed the chinese characters; the default help should better be in English for a global audience.